examples/README.md

```diff
@@ -0,0 +1,40 @@
+# Starting containerlab labs
+
+## Example labs topologies
+Those example labs all have the same topology, but with different vendors.  We have two routers, east and west, connected to each other, and with two subscribers each.
+
+```
+ ┌──────────────────────┐                    ┌──────────────────────┐
+ │                      │eth1            eth1│                      │
+ │     router-west      ├────────────────────┤     router-east      │
+ │                      │                    │                      │
+ │ mgmt-ip:172.20.20.31 │                    │ mgmt-ip:172.20.20.21 │
+ └───┬──────────────┬───┘                    └───┬──────────────┬───┘
+     │  eth2    eth3│                            │eth3    eth2  │
+     │              └──────────┐      ┌──────────┘              │
+     │                         │      │                         │
+     │  eth1                   │      │                   eth1  │
+ ┌───┴──────────────────┐      │      │      ┌──────────────────┴───┐
+ │                      │      │      │      │                      │
+ │  subscriber-west-1   │      │      │      │  subscriber-east-1   │
+ │                      │      │      │      │                      │
+ │ mgmt-ip:172.20.20.32 │      │      │      │ mgmt-ip:172.20.20.22 │
+ └──────────────────────┘      │      │      └──────────────────────┘
+                               │      │
+ ┌──────────────────────┐      │      │      ┌──────────────────────┐
+ │                      │eth1  │      │  eth1│                      │
+ │  subscriber-west-2   ├──────┘      └──────┤  subscriber-east-2   │
+ │                      │                    │                      │
+ │ mgmt-ip:172.20.20.33 │                    │ mgmt-ip:172.20.20.23 │
+ └──────────────────────┘                    └──────────────────────┘
+```
+
```

Comment:
It would be good to reference nfv-test-api here.

Reply:
Done in https://github.com/inmanta/containerlab-vms/pull/5/commits/a284bd62f31982ab0ca854fc002836a7f83b1632